src/main/java/ch/njol/skript/command/Commands.java

```diff
@@ -311,7 +311,7 @@ static boolean handleEffectCommand(final CommandSender sender, String command) {
 	
 	@SuppressWarnings("null")
 	private final static Pattern commandPattern = Pattern.compile("(?i)^command /?(\\S+)\\s*(\\s+(.+))?$"),
-			argumentPattern = Pattern.compile("<\\s*(?:(.+?)\\s*:\\s*)?(.+?)\\s*(?:=\\s*(" + SkriptParser.wildcard + "))?\\s*>");
+			argumentPattern = Pattern.compile("<\\s*(?:([^>]+?)\\s*:\\s*)?(.+?)\\s*(?:=\\s*(" + SkriptParser.wildcard + "))?\\s*>");
```

Comment:
As long as this has been properly tested LGTM

Reply:
I have tested the following things:

**Test 1.**

```
command /test <var:text> <var2:text>:
    trigger:
        send "%{_var}% - %arg-1% | %{_var2}% - %arg-2%"
```

`/test argument_1 argument_2`

![billede](https://user-images.githubusercontent.com/75088349/173098891-c5ecb256-3d4b-4a45-a40c-fc5782c94a22.png)


**Test 2.**

```
command /test <var:text> [<var2:text>]:
    trigger:
        send "%{_var}% - %arg-1% | %{_var2}% - %arg-2%"
```

`/test argument_1`
![billede](https://user-images.githubusercontent.com/75088349/173099262-4e5f7d73-8dfb-45d6-8cb6-50902ffde37e.png)

`/test argument_1 argument_2`
![billede](https://user-images.githubusercontent.com/75088349/173099367-1c5a6d69-0f6b-4f4a-a792-f04f087df05a.png)

**Test 3.**

```
command /test <text> <var2:text>:
    trigger:
        send "%{_var}% - %arg-1% | %{_var2}% - %arg-2%"
```

`/test argument_1 argument_2`
![billede](https://user-images.githubusercontent.com/75088349/173099693-5daa546d-e570-4e11-add1-cb5ac4c7e3ff.png)

**Test 4.**

```
command /test <var:text> <text>:
    trigger:
        send "%{_var}% - %arg-1% | %{_var2}% - %arg-2%"
```

`/test argument_1 argument_2`
![billede](https://user-images.githubusercontent.com/75088349/173099950-7e49b247-7219-4e0b-b557-afaf07afc336.png)

**Test 5.**

```
command /test <var:text>:
    trigger:
        send "%{_var}% - %arg-1%"
```

`/test argument_1`
![billede](https://user-images.githubusercontent.com/75088349/173100547-109fba44-4842-4e77-890e-c8230b76c4d0.png)


**Test 6.**

```
command /test [<var:text>]:
    trigger:
        send "%{_var}% - %arg-1%"
```

`/test argument_1`
![billede](https://user-images.githubusercontent.com/75088349/173100867-7db02bc9-2399-4d5b-953a-e58a80821614.png)

'/test'
![billede](https://user-images.githubusercontent.com/75088349/173100908-b1eead08-b997-4868-9744-488d9a285e43.png)

**Test 7.**

```
command /test <text>:
    trigger:
        send "%{_var}% - %arg-1%"
```

`/test argument_1`
![billede](https://user-images.githubusercontent.com/75088349/173101254-bcb58047-a427-4b5e-a675-f816db6cfead.png)

**Test 8.**

```
command /test <var:text> <var2:text> <var3:text>:
    trigger:
        send "%{_var}% - %arg-1% | %{_var2}% - %arg-2% | %{_var3}% - %arg-3%"
```

`/test argument_1 argument_2 argument_3`
![billede](https://user-images.githubusercontent.com/75088349/173101696-ecf64873-fdbd-4162-b85f-ed4a1df37d31.png)

I have tested all of them with both number and text as types.

All of the tests run as expected.